index.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+<center>
+  <h2>Wanna hear from our newsletter?</h2>
+  <form>
+    <input type="email">
+    <br>
```

Comment:
I know it's just a first throw of a landing page but I think we should take care to have nice code also. I think we should solve this without the `<br>`, but my css is not that good that I've a better solution in my head, sorry. I can look that up tonight when the kid is sleeping :)

Reply:
Good point! I never remmeber if using breaklines is in or not 🙈 

I'll find a better solution